packages/react-sdk/src/components/Whiteboard/Element/ConnectedElement.tsx

```diff
@@ -72,11 +76,34 @@ export const ConnectedElement = ({
       }
 
       return (
-        <ImageDisplay
-          baseUrl={widgetApi.widgetParameters.baseUrl}
-          {...element}
-          {...otherProps}
-        />
+        <Suspense
```

Comment:
Can we move this into the `elements/image` module?
Why not move it into the `ImageDisplay` and move the current `ImageDisplay` to a new component.

At least the logic should all be in `elements/image` and not here in my opinion.